xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -323,14 +325,13 @@ bool CDRMUtils::FindCrtcs()
                   object->crtc->crtc_id, strerror(errno));
         drmModeFreeCrtc(object->crtc);
         delete object;
-        continue;
       }
 
-      m_crtcs.emplace_back(object);
-
       if (object->crtc->crtc_id == m_encoder->encoder->crtc_id)
         m_orig_crtc = object;
```

Comment:
```suggestion
      if (object && object->crtc->crtc_id == m_encoder->encoder->crtc_id)
        m_orig_crtc = object;
```

Since `object` might have been deleted above, I think this needs an extra check here.